### DISCUSSION TITLE
inpaint/outpaint default too slow after update to 2.5.2

### DISCUSSION THREAD

**Discussion by anikison** [lllyasviel/Fooocus#3398] [Q&A] [answered]
Guys,

Are there any changes on inpaint/outpaint default 2.5.2?  my image is processing like forever when i try to paint some area and use default.

Meanwhile, when i add details, content or expand any direction sounds like normal..

no errors on output window.

[Fooocus] Image processing ...
Upscaling image with shape (768, 950, 3) ...

**Comment by mashb1t** [comment, marked as answer]
There have been no changes to the inpainting functionality in v2.5.2, see [release notes](https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.2). Please consider checking the task manager to identify bottlenecks such as full VRAM incl. shared VRAM usage and offloading to swap.
